src/color.scss

```diff
@@ -0,0 +1,118 @@
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
+
+/* stylelint-disable no-descending-specificity */
+
+// Handle highlighting the border during focus and invalid state
+@mixin border-highlight {
+  &:before {
+    border-bottom-color: transparent;
+  }
+}
+
+.wrapper {
+  border-color: transparent;
+}
+
+.inputElement-helpText {
+  color: var(--ds-auro-input-text-color);
+}
+
+input {
+  background-color: transparent;
+  caret-color: var(--ds-auro-input-text-color);
+
+  &::placeholder {
+    color: transparent;
+  }
+
+  &:focus {
+    &::placeholder {
+      color: var(--ds-auro-input-text-color);
+    }
+  }
```

Comment:
placeholder text should probably have it's own tier 3 token which defaults to transparent and then change the value during focus.